spec/unit/vcn_list_spec.rb

```diff
@@ -0,0 +1,96 @@
+# Copyright (c) 2017 Oracle and/or its affiliates. All rights reserved.
```

Comment:
The list vcn and subnet changes are getting pulled in... maybe a rebase is needed?